### PR TITLE
Updated daily spending case study for Lightning

### PR DESCRIPTION
**Pull request by sbddesign**
I updated the Daily Spending case study for Lightning. Proposed fix for #615.

⚡️ [Deploy Preview](https://deploy-preview-631--sad-borg-390916.netlify.app/guide/case-studies/cloud-backup/)

### CODE REVIEW

**Comment by danielnordh** [comment]
Thanks for jumping on this @sbddesign.

Most of my feedback is on the new screens themselves, do you have a Figma that I could comment on directly?
I think that might be more efficient.

**Comment by sbddesign** [comment]
> Most of my feedback is on the new screens themselves, do you have a Figma that I could comment on directly?

@danielnordh [FIgma Link](https://www.figma.com/file/dXSeWwzKoVvS0Xm9VT9HuJ/Daily-Spending-Case-Study---Bitcoin-Design-Guide?node-id=0%3A1)

**Comment by danielnordh** [comment]
OK, left some comments on the Figma.

To try and summarize:
- While the previous screenshots are 'only wireframes, they attempted to be very precise in the actual information they contained, how they fit together, and what flows started and ended where
- Even though these new screens are 'higher fidelity' visually, they feel less precise and informative in terms of information details and flows
- It is not obvious what the Lightning default of this version does or doesn't enable for the user and how it might differ from the previous case

**Comment by sbddesign** [comment]
Thanks for the feedback, @danielnordh. I did a good bit of work inside of the Figma document. Would you be able to look at that and let me know what you think of my updates before I put the new images into this PR? I added some commentary in their, as well.

> It is not obvious what the Lightning default of this version does or doesn't enable for the user and how it might differ from the previous case

The Lightning default of this version enables the product to actually function as a daily spending wallet. The current case study outlines a scenario where an on-chain wallet is being used for daily spending. IMHO, that's not feasible due to the on-chain transaction confirmation time.

**Comment by sbddesign** [comment]
I have made several updates to this PR. It now includes a much more extensive prototype as well as some simplifications of the UX designs.

> The fact that it is Lightning only is not explained.

@danielnordh I'm kind of operating here under our philosophy of wanting to abstract away the difference between Lightning and on-chain so the user doesn't have to worry about it.

This may seem odd, but I don't think it's unachievable (Muun has pioneered this and I'm sure many will follow). I added some notes in this case study to the design and technical considerations in an attempt to make this clearer to the reader. From the developer's perspective, a submarine swap provider could be integrated with the wallet so that the user can send to and receive from on-chain addresses using their Lightning wallet. From the user's perspective, it's easy enough to use that they don't need to worry about it. Some QR codes (Lightning) look different to them than other QR codes (on-chain), but they all seem to work with this simplified bitcoin wallet.

**Comment by danielnordh** [comment]
Thanks @sbddesign, this looks good to me.
Just added a minor comment in the prototype about a typo.

**Review by danielnordh — APPROVED** [review]
LGTM

**Comment by sbddesign** [comment]
> Just added a minor comment in the prototype about a typo.

Fixed, thanks!

File: guide/case-studies/cloud-backup.md [hunk 1]
```diff
@@ -43,85 +20,172 @@ https://www.figma.com/community/file/968416729557947210
...
+The recovery flow guides the user through restoring their wallet on a new device. For example, the user creates their wallet with their smartphone, then upgrades to a new phone, and wants to "transfer" the wallet to their new phone.
 
 <div class="image-slide-gallery">
 
-{% for item in page.importImages %}
+{% include picture.html
+   image = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-restore-menu.png"
+   retina = "/assets/images/guide/case-studies/cloud-backup/daily-spending-wallet-restore-menu@2x.png"
+   width = 250
+   height = 600
```
Comment (GBKS) [inline guide/case-studies/cloud-backup.md:137]: Height for these images should also be 541 to match the proportions of the image files.

The image files should ideally also be 250x541, they are currently 375*812.

**Comment by GBKS** [comment]
Great update. Just had a minor nitpick about image sizes & width/height attributes.

There's a "Manual restore" flow in the prototype that is not in the page or in the rest of the guide. Something we can consider adding to the onboarding section.

File: guide/case-studies/cloud-backup.md (hunk 1 above)
Comment (sbddesign) [reply guide/case-studies/cloud-backup.md]: Fixed

**Review by GBKS — APPROVED** [review]
ACK.

**Pull request merged**